_ingest-pipelines/processors/text-chunking.md

```diff
@@ -157,159 +157,11 @@ The response confirms that, in addition to the `passage_text` field, the process
-
-```json
-{
-  "docs": [
-    {
-      "doc": {
-        "_index": "testindex",
-        "_id": "1",
-        "_source": {
-          "passage_chunk_embedding": [
-            {
-              "knn": [...]
-            },
-            {
-              "knn": [...]
-            },
-            {
-              "knn": [...]
-            }
-          ],
-          "passage_text": "This is an example document to be chunked. The document contains a single paragraph, two sentences and 24 tokens by standard tokenizer in OpenSearch.",
-          "passage_chunk": [
-            "This is an example document to be chunked. The document ",
-            "The document contains a single paragraph, two sentences and 24 ",
-            "and 24 tokens by standard tokenizer in OpenSearch."
-          ]
-        },
-        "_ingest": {
-          "timestamp": "2024-03-20T03:04:49.144054Z"
-        }
-      }
-    }
-  ]
-}
-```
-
-Once you have created an ingest pipeline, you need to create an index for ingestion and ingest documents into the index. To learn more, see [Step 2: Create an index for ingestion]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/#step-2-create-an-index-for-ingestion) and [Step 3: Ingest documents into the index]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/#step-3-ingest-documents-into-the-index) of the [neural sparse search documentation]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/).
-
-## Cascaded text chunking processors
-
-You can chain multiple chunking processors together. For example, to split documents into paragraphs, apply the `delimiter` algorithm and specify the parameter as `\n\n`. To prevent a paragraph from exceeding the token limit, append another chunking processor that uses the `fixed_token_length` algorithm. You can configure the ingest pipeline for this example as follows:
-
-```json
-PUT _ingest/pipeline/text-chunking-cascade-ingest-pipeline
-{
-  "description": "A text chunking pipeline with cascaded algorithms",
-  "processors": [
-    {
-      "text_chunking": {
-        "algorithm": {
-          "delimiter": {
-            "delimiter": "\n\n"
-          }
-        },
-        "field_map": {
-          "passage_text": "passage_chunk1"
-        }
-      }
-    },
-    {
-      "text_chunking": {
-        "algorithm": {
-          "fixed_token_length": {
-            "token_limit": 500,
-            "overlap_rate": 0.2,
-            "tokenizer": "standard"
-          }
-        },
-        "field_map": {
-          "passage_chunk1": "passage_chunk2"
-        }
-      }
-    }
-  ]
-}
-```
-{% include copy-curl.html %}
+Once you have created an ingest pipeline, you need to create an index for ingestion and ingest documents into the index. To learn more, see [Text chunking]({{site.url}}{{site.baseurl}}/search-plugins/text-chunking/).
```

Comment:
The text after the comma in the first sentence is a bit redundant. Can we simplify?

Reply:
Creating an index is a separate step so I think separating the two makes sense, even though "ingest" appears in both parts of the sentence.